### PR TITLE
Add filter flag

### LINKED ISSUE #103: Provide an option for a single operator test
Use a flag to allow for single operator testing.

### PR DESCRIPTION
**Pull request by mrhillsman**
Allow testing a single operator. You will need to know the specific package name to pass to filter. --list-packages can be used to obtain the package name.
Resolves #103

Signed-off-by: Melvin Hillsman <mrhillsman@gmail.com>

### CODE REVIEW

**Review by acmenezes — CHANGES_REQUESTED** [review]
This looks pretty good! I wonder if we could call `filter` something more meaningful like `packages` for example.

File: internal/operator/subscription.go [hunk 1]
```diff
@@ -85,6 +85,23 @@ func (c operatorClient) DeleteSubscription(ctx context.Context, name string, nam
...
+	if len(filter) > 0 {
+		for _, f := range filter {
+			for _, pkgm := range tmppkgmlist.Items {
+				if pkgm.Name == f {
+					list.Items = append(list.Items, pkgm)
+				}
+			}
+		}
+	} else {
+		list.Items = tmppkgmlist.Items
```
Comment (acmenezes) [inline internal/operator/subscription.go:103]: One thing that came to me is that if we should have some sort of output to tell the user that the package name provided doesn't exist on the desired catalog source. Not sure how relevant it is though.

**Review by yashoza19 — APPROVED** [review]
/approve

File: internal/operator/subscription.go (hunk 1 above)
Comment (mrhillsman) [reply internal/operator/subscription.go:103]: Yep I opened issues that are probably relevant to this but let me know if not; #116 and #117 @acmenezes

Comment (acmenezes) [reply internal/operator/subscription.go:103]: Yes @mrhillsman , that works. I guess we can even address this in more than one PR if needed.

Comment (mrhillsman) [reply internal/operator/subscription.go:103]: @acmenezes yep I believe that is best as I did not want to make this PR too large and still need to work out the logic while filtering can be done for now in some capacity with this one.

**Review by skattoju — APPROVED** [review]

**Pull request merged**